main/core/src/main/java/org/cryptomator/webdav/WebDavServer.java

```diff
@@ -100,6 +112,32 @@ public ServletLifeCycleAdapter createServlet(final Path workDir, final boolean c
 		}
 	}
 
+	/**
+	 * Tries to form a similar string using the regular latin alphabet.
+	 * 
+	 * @param string
+	 * @return a string composed of a-z, A-Z, 0-9, and _.
+	 */
+	public static String normalize(String string) {
```

Comment:
I tried that, but it didn't work at all. I think the best course of action is to leave this method here and possibly refine it if we want to allow a broader spectrum of names.